src/server/address-format-reviver.ts

```diff
@@ -0,0 +1,32 @@
+/*
+A reviver is a function that prescribes how the value originally produced by JSON.parse is transformed, before being returned.
+*/
+
+import { formatEthereumAddress } from "speedomatic";
+
+export interface StringToBoolMap {
+  [key: string]: boolean;
+}
+
+export const inputsExpectedAsAddress: StringToBoolMap = {
+  account: true,
+  category: true,
+  creator: true,
+  designatedReporter: true,
+  marketID: true,
+  marketIDs: true,
+  reporter: true,
+  reportingWindow: true,
+  result: true,
```

Comment:
result? I don't know if this is necessary as a parameter to a getter

Reply:
ic -- it is the name of the variable in the "callback" from the definitions. removing